### PR TITLE
Handle extra object literal properties (Part IV)

### PR DESCRIPTION
**Pull request by DanielRosenwasser**
This is part of sweep of work to address Microsoft/TypeScript#4081.

See Part I: #5151.
See Part II: #5233.
See Part III: #5346.

See Part V: #5645.

### CODE REVIEW

**Comment by DanielRosenwasser** [comment]
@danquirk @RyanCavanaugh

File: jquery.gridster/gridster.d.ts [hunk 1]
```diff
@@ -248,3 +248,7 @@ interface Gridster {
 	**/
 	disable(): Gridster;
 }
+
+interface JQuery {
+    data(key: "gridster"): Gridster;
```
Comment (DanielRosenwasser) [inline jquery.gridster/gridster.d.ts]: I actually am not 100% whether or not this was the correct thing to do.

File: parse/parse.d.ts [hunk 2]
```diff
@@ -420,24 +419,49 @@ declare module Parse {
...
+        interface CreateOptions extends BackboneStyleOptions, WaitOption, SilentOption, UseMasterKeyOption {
+        }
+
+        interface FetchOptions extends BackboneStyleOptions, SilentOption, UseMasterKeyOption { }
+
+        interface RemoveOptions extends SilentOption { }
 
+        interface ResetOptions extends SilentOption { }
+
+        interface SortOptions extends SilentOption { }
```
Comment (danquirk) [inline parse/parse.d.ts]: :( These are all just empty besides optional at?

File: parse/parse.d.ts [hunk 3]
```diff
@@ -588,6 +612,13 @@ declare module Parse {
         withinRadians(key: string, point: GeoPoint, maxDistance: number): Query;
     }
 
+    namespace Query {
+        interface CountOptions extends BackboneStyleOptions, UseMasterKeyOption { }
+        interface FindOptions extends BackboneStyleOptions, UseMasterKeyOption { }
+        interface FirstOptions extends BackboneStyleOptions, UseMasterKeyOption { }
+        interface GetOptions extends BackboneStyleOptions, UseMasterKeyOption { }
```
Comment (danquirk) [inline parse/parse.d.ts]: these are all exactly the same?

**Comment by danquirk** [comment]
:+1:

**Comment by DanielRosenwasser** [comment]
@vvakame can you take a look?

**Comment by DanielRosenwasser** [comment]
Actually, hold off on pulling this in please.

**Comment by vvakame** [comment]
:+1:

**Comment by DanielRosenwasser** [comment]
@vvakame I think we can pull this in now.

Also @borisyankov any chance I can get merging privileges?

**Comment by borisyankov** [comment]
@DanielRosenwasser sounds like a good idea. You are in ;)

**Comment by DanielRosenwasser** [comment]
Thanks!

**Pull request merged**